config/versions.json

```
@@ -1,5 +1,5 @@
 {
     "$schema": "https://raw.githubusercontent.com/ACCESS-NRI/schema/main/au.org.access-nri/model/deployment/config/versions/3-0-0.json",
     "spack": "0.22",
-    "spack-packages": "SOME_SPECIFIC_TAG"
+    "spack-packages": "2025.04.000"
```

```suggestion
    "spack-packages": "2025.04.001"
```
I have pushed a new tag. This needs to be updated to pick up the changes to `spack-packages`